rcfc/button.py

```diff
@@ -5,16 +5,19 @@
 from rcfc.server import register_post, register_post_with_state
 
 
-def simple(text):
+def simple(text, group=None):
```

Comment:
I'm not really a fan of just making this how group functionality is enabled. Seems like any more button types are going to need this as well, but I couldn't see a good way to wrap these decorator definitions in a decorator to get the intended functionality. With this, it feels like there would need to be a test for each type of button to make sure that grouping works.

Reply:
I'd like to see decorators stack:

```python
@grouping("Lights")
@button.simple("Turn on all lights")
def blah_blah_blah(): pass
```

Take a look at http://click.pocoo.org/5/ to get an idea of how to do it.

Reply:
I'm not entirely sure how to set it up for that, since the items that need to be modified are within the inner wrapper function. That is what I wanted to do, but I couldn't figure out how to accomplish that.